src/javascript/app/App/Components/Layout/Footer/toggle-positions.jsx

```diff
@@ -1,8 +1,7 @@
 import classNames        from 'classnames';
 import PropTypes         from 'prop-types';
 import React             from 'react';
-import { Icon }          from 'Assets/Common';
-import { IconPositions } from 'Assets/Footer';
+import Icon              from 'Assets/icon.jsx';
```

Comment:
can you remove the extra spaces ?

Reply:
@kellybinary fixed this and the timepicker